features/emp-selector/EmpSelector.tsx

```diff
@@ -1,6 +1,8 @@
 import styled from "styled-components";
-import Box from "@material-ui/core/Box";
+import { Box } from "@material-ui/core";
 import { withStyles } from "@material-ui/core/styles";
+import useMediaQuery from "@material-ui/core/useMediaQuery";
```

Comment:
nit: can't you import `useMediaQuery` in same struct as `Box`?

Reply:
ye, you right. will go through all the components and look for repeated imports. I suspect there are more than just this one.